src/main/java/games/strategy/engine/framework/headlessGameServer/AvailableGames.java

```diff
@@ -99,18 +99,23 @@ private static void populateAvailableGames(final Map<String, URI> availableGames
     return Arrays.asList(files);
   }
 
-  private static void populateFromDirectory(final File mapDir, final Map<String, URI> availableGames,
-      final Set<String> availableMapFolderOrZipNames, final Set<String> mapNamePropertyList) {
+  private static void populateFromDirectory(
+      final File mapDir,
+      final Map<String, URI> availableGames,
+      final Set<String> availableMapFolderOrZipNames,
+      final Set<String> mapNamePropertyList) {
     final File games = new File(mapDir, "games");
     if (!games.exists()) {
       // no games in this map dir
       return;
     }
-    for (final File game : games.listFiles()) {
-      if (game.toURI() != null && game.isFile() && game.getName().toLowerCase().endsWith("xml")) {
-        final boolean added = addToAvailableGames(game.toURI(), availableGames, mapNamePropertyList);
-        if (added) {
-          availableMapFolderOrZipNames.add(mapDir.getName());
+    if (games.listFiles() != null) {
```

Comment:
Wouldn't it be better to check if games.isDirectory() and add that to the if statement right above?